src/components/NavBar/new/MainNavbar/index.js

```diff
@@ -25,19 +25,23 @@ import useWindowSize from "../../../../helpers/customHooks/useWindowSize";
 const useStyles = makeStyles(theme => ({
   input: {
     marginLeft: theme.spacing(1),
+    color: "#3e5060",
+    letterSpacing:"0.5px",
     flex: 1,
     width:"92%",
     [theme.breakpoints.down("md")]:{
       width:"80%"
     }
   },
   root: {
-    backgroundColor: theme.palette.grey[100],
+    backgroundColor: "#f3f6f9",
```

Comment:
Use `theme.palette.primary` colors and use Material ui theme palette colors instead of hardcoded hex

Reply:
Ok